e2e/atlas/ldap_test.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 const (
-	PENDING  = "PENDING"
-	HOSTNAME = "2.tcp.ngrok.io"
+	pending  = "pending"
```

Comment:
Actually here there is a mistake:
```suggestion
	pending  = "PENDING"
```

Reply:
when you submit this change please schedule the e2e test for ldap on evergreen

Reply:
done